Example_App/Example - optimove.json

```diff
@@ -1,4 +1,5 @@
 {
     "optimoveCredentials": "YOUR_OPTIMOVE_CREDENTIALS",
-    "optimoveMobileCredentials": "YOUR_OPTIMOVE_MOBILE_CREDENTIALS"
+    "optimoveMobileCredentials": "YOUR_OPTIMOVE_MOBILE_CREDENTIALS",
+    "inAppConsentStrategy" : "YOUR_IN_APP_CONSENT_STRATEGY"
```

Comment:
In the example file, should we put the 'disabled' value?

Reply:
I'm sorry I'm not sure I understand the question, where should we put disabled?

Reply:
do you mean to put 'disabled' value to all the properties that should be missing?

Reply:
I changed "YOUR_IN_APP_CONSENT_STRATEGY" to in-app-disabled